recipe/meta.yaml

```diff
@@ -9,51 +9,45 @@ source:
   sha256: 36f2132482a2eb9910ff9f760f0e61168aee874ad473a187cd8e5f7db2d2e617
 
 build:
-  number: 1
-  skip: True  # [win32 or (win and py<=35)]
-  features:
-    - vc14  # [win and (py35 or py36)]
+  number: 2
+  skip: True  # [win and vc<14]
 
 requirements:
   build:
-    - python  # [win]
     - cmake  # [win]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
     - antlr >=2.7.7,<3
     - bison  # [not win]
     - texinfo  # [not win]
     - flex  # [not win]
     - m4  # [not win]
-    - curl >=7.44.0,<8
-    - expat 2.2.*
-    - gsl >=2.2,<2.3
-    - hdf5 1.10.1
-    - krb5 1.14.*  # [not win]
-    - libnetcdf 4.6.*
+    - curl
+    - expat
+    - gsl
+    - hdf5
+    - krb5  # [not win]
+    - libnetcdf
     - udunits2
-    - zlib 1.2.11
-    - vc 14  # [win and (py35 or py36)]
+    - zlib
   run:
-    - curl >=7.44.0,<8
-    - expat 2.2.*
-    - gsl >=2.2,<2.3
-    - hdf5 1.10.1
-    - krb5 1.14.*  # [not win]
-    - libnetcdf >=4.6
+    - curl
+    - expat
+    - gsl
+    - hdf5
+    - krb5  # [not win]
+    - libnetcdf
     - udunits2
-    - libgcc  # [not win]
-    - openblas 0.2.20|0.2.20*  # [not win]
     - esmf  # [not win]
-    - vc 14  # [win and (py35 or py36)]
 
 test:
   source_files:
     - data/in.cdl
   commands:
     - ncks --help
-    - ncks -M "http://tds.marine.rutgers.edu/thredds/dodsC/roms/espresso/2013_da/his/ESPRESSO_Real-Time_v2_History_Best"
     - ncap2 --help
-    - conda inspect linkages -p $PREFIX $PKG_NAME  # [not win]
-    - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
+    - ncks -M "http://tds.marine.rutgers.edu/thredds/dodsC/roms/espresso/2013_da/his/ESPRESSO_Real-Time_v2_History_Best"
```

Comment:
I presume we've moved on but just in case...

A colleague of mine tested this on his Mac and it worked fine as far as we could tell, something like:
```
netcdf ESPRESSO_Real-Time_v2_History_Best {
  dimensions:
    boundary = 4 ;
    eta_psi = 81 ;
    eta_rho = 82 ;
    eta_u = 82 ;
    eta_v = 81 ;
    ocean_time = UNLIMITED ; // (24 currently)
    s_rho = 36 ;
    s_w = 37 ;
    time = 45516 ;
    tracer = 2 ;
    xi_psi = 129 ;
    xi_rho = 130 ;
    xi_u = 129 ;
    xi_v = 130 ;
...
    :DODS_EXTRA.Unlimited_Dimension = "ocean_time" ;
    :EXTRA_DIMENSION.N = 36 ;
} // group /
```

Reply:
Thanks. How about the parts in https://github.com/conda-forge/nco-feedstock/blob/master/recipe/run_test.sh#L32-L35

(Although we may need to wait for #70 to test this again.)

Reply:
Hmm, sorry for the confusion.  I didn't realize that's what you meant. I was going off of the commit messages.

I don't want to pester my colleague too much about this stuff so I think I'll need to wait and test this again once we see what happens with #70

Reply:
> Hmm, sorry for the confusion.

NP. I was not clear at all in my first message :smile: 

> I don't want to pester my colleague too much about this stuff so I think I'll need to wait and test this again once we see what happens with #70

Yep. Don't worry about it until we have everything sorted all in #70. Thanks!